src/directives/formly-form.test.js

```diff
@@ -952,5 +952,31 @@ describe('formly-form', () => {
       expect(listener).to.have.been.called
       expect(expression).to.have.been.called
     })
+
+    it(`should setup any watchers specified on a fieldgroup`, () => {
+      scope.model = {}
+
+      const listener = sinon.spy()
+      const expression = sinon.spy()
+
+      scope.fields = [{
+        watcher: [{
+          listener: '',
+          expression: '',
+        }, {
+          listener,
+          expression,
+        }],
+        key: 'group',
```

Comment:
Yeah, that would indicate to me that anyone using field groups without specifying a `key` on the field group object will get that error when compiling. Could you investigate it further so the key is not required?